README2.md

```diff
@@ -0,0 +1 @@
+this is an update to ES 7 - updating the index/bulk to write
```

Comment:
instead of readme2 file, add a CHANGELOG file with this information

Reply:
removed README2 - will talk about CHANGELOG